### PR TITLE
Paper: Appendix

### PR DESCRIPTION
**Pull request by beyond2void**

### CODE REVIEW

**Comment by beyond2void** [comment]
image

current appendix view is like this.

image

code view.

Because build-in appendix commands have to work with table of contents, 
so I added appendix and its style manually.

I think later we could also add some figures inside appendix,
so that our paper looks more clean.

File: paper/appendix.tex [hunk 1]
```diff
@@ -0,0 +1,1042 @@
+{\huge \textbf{Appendix A: User Study}}\\\\\\\\  % Minye
+{\large \textbf{A.1: Interview Tasks }} \\\\
+\section{Interview Tasks}\label{appendix:tasks}
+This part is about interview tasks we prepared to conduct our user study.
```
Comment (PaulBredl) [inline paper/appendix.tex:4]: Please also maybe note that the tasks were modified and also that the third task was only added after the first user study

**Review by PaulBredl — DISMISSED** [review]
Other than the one comment looks good. Does your styling of the code also apply to the other code blocks?

**Comment by beyond2void** [comment]
> Other than the one comment looks good. Does your styling of the code also apply to the other code blocks?

Yes, when you put the code block inside 
```
/begin{lstlisting}
/end{lstlisting}
```
How do you think of this style?
I can make other codes in our paper consistent if you think It's good.

**Comment by PaulBredl** [comment]
> > Other than the one comment looks good. Does your styling of the code also apply to the other code blocks?
> 
> Yes, when you put the code block inside
> 
> ```
> \begin{lstlisting}
> \end{lstlisting}
> ```
> 
> How do you think of this style? I can make other codes in our paper consistent if you think It's good.

Yeah pls make it consistent

**Comment by beyond2void** [comment]
截屏2023-10-16 13 15 37
Current view of code listing

**Pull request merged**